service-level-indicators/spark-retrieval-success-rate.md

```diff
@@ -0,0 +1,383 @@
+# Spark Request-Based (Non-Committee) Global Retrieval Success Rate <!-- omit from toc -->
+
+- [Meta](#meta)
+  - [Document Purpose](#document-purpose)
+  - [Versions](#versions)
+  - [Support, Questions, and Feedback](#support-questions-and-feedback)
+- [TL;DR](#tldr)
+- [Spark Protocol](#spark-protocol)
+  - [Deal Ingestion](#deal-ingestion)
+  - [Task Sampling](#task-sampling)
+    - [**Which of the tasks in the Round Retrieval Task List should each Checker perform?**](#which-of-the-tasks-in-the-round-retrieval-task-list-should-each-checker-perform)
+    - [**How does the protocol orchestrate the Spark Checkers to perform different tasks from each other at random  such that each Task is completed by at least $m$ checkers?**](#how-does-the-protocol-orchestrate-the-spark-checkers-to-perform-different-tasks-from-each-other-at-random--such-that-each-task-is-completed-by-at-least-m-checkers)
+    - [**How many tasks should each checker do per round? What value is given to $k$?**](#how-many-tasks-should-each-checker-do-per-round-what-value-is-given-to-k)
```

Comment:
Why are these bold?

Reply:
They were bolded because they were headings with bold down below. (and the toc is autogenerated)  That said, I just kept them bold now and removed their heading.  Resolving